packages/ember/tests/routing/router_service_test/isActive_test.js

```diff
@@ -168,5 +168,23 @@ moduleFor(
           assert.deepEqual(qp.queryParams, { sort: 'ascending' });
         });
     }
+
+    ['@test RouterService#isActive calls setupRouter to ensure router is initialized'](assert) {
+      assert.expect(1);
+
+      return this.visit('/').then(() => {
+        let setupRouterCallCount = 0;
+        let originalSetupRouter = this.routerService._router.setupRouter;
+
+        this.routerService._router.setupRouter = function () {
+          setupRouterCallCount++;
+          return originalSetupRouter.call(this);
+        };
+
+        this.routerService.isActive('parent.child');
```

Comment:
what happened when accessing isActive previously?

from the test here, it's not clear.

is there a way we can assert behavior without overriding setupRouter?
What behavior are we fixing here?

(as is, this test asserts an implementation detail, and implementation details are less important to our users)